doc/content/devices/device-claiming/make-device-claimable.md

```diff
@@ -5,42 +5,38 @@ weight: 1
 distributions: ["Enterprise", "Cloud"]
 --- 
 
-This guide explains the process of making a device claimable 
+{{< cli-only >}}
 
-<!--more-->
+This guide explains the process of making a device claimable.
 
-{{< cli-only >}}
+<!--more-->
 
 ## Prerequisites
 
-1. Access to The Things Industries Cloud. [Contact The Things Industries](mailto:cloud@thethingsindustries.com) to get onboarded.
+1. Access to The Things Industries Cloud. [Contact The Things Industries](mailto:cloud@thethingsindustries.com) to get onboarded
 3. An application in The Things Industries Cloud. [See instructions]({{< ref "/integrations/adding-applications" >}})
 
 ## Register Devices
 
-In order to be claimable the device needs to be registered on The Things Industries Join Server. This can be done in multiple ways.
+In order to be claimable, the device needs to be registered on The Things Industries Join Server. This can be done in multiple ways.
 
 ### Registered on The Things Industries Join Server Only
 
-1. With root keys and claim authentication code  
-  Follow the [Register Devices]({{< ref "/getting-started/cloud-hosted/tti-join-server/register-devices" >}}) guide.
+1. With root keys and claim authentication code. Follow the [Register Devices]({{< ref "/getting-started/cloud-hosted/tti-join-server/register-devices" >}}) guide.
```

Comment:
This needs to stay the way it was, because "With root keys and claim authentication code" isn't a full sentence.

```suggestion
1. With root keys and claim authentication code
- Follow the [Register Devices]({{< ref "/getting-started/cloud-hosted/tti-join-server/register-devices" >}}) guide.
```